class.jetpack.php

```diff
@@ -6775,6 +6775,7 @@ public static function dashboard_widget_footer() {
 		<footer>
 
 		<div class="protect">
+			<h4>Jetpack Protect</h4>
```

Comment:
I wonder if these should be h4, given that we have h3 afterwards to show the numbers. It may be best to keep things hierarchical.

Reply:
Hm, very true. There is already an `h2` higher on the page, so should these be `h3` / `h4`? That would look like the below:

<img width="252" alt="Screen Shot 2020-02-04 at 8 53 52 PM" src="https://user-images.githubusercontent.com/4930129/73804057-93472f00-4790-11ea-8b56-ffed3b7c6b1e.png">

In this case, the number is less prominent than before -- not sure if that aligns well with the intended goal of the widget? Maybe the number should be a non-header tag, so that it can be styled separately as a larger font?

Reply:
> should these be h3 / h4?

I think we could use h3 for that heading, and not use any heading tag for the number. Instead we could style to look just like before with CSS.

The stylesheet used in that widget is here:
https://github.com/Automattic/jetpack/blob/f2bcdc78dd3ba56b9b07bde603f4d2005464ded6/_inc/client/dashboard-widget/style.scss#L1